master/setup.py

```diff
@@ -166,16 +170,17 @@ def define_plugin_entries(groups):
         "buildbot.steps.package.deb",
         "buildbot.steps.package.rpm",
         "buildbot.steps.source",
-        "buildbot.test",
+        "buildbot.test.util",
```

Comment:
If tests are not included, why do we want to still include `buildbot.test.{util,fake}`?